tests/src/Kernel/Access/OgEntityAccessTest.php

```diff
@@ -133,6 +177,15 @@ protected function setUp() {
       ->grantPermission($this->randomMachineName())
       ->save();
 
+    $this->ogAdminRole = OgRole::create();
+    $this->ogAdminRole
+      ->setId($this->randomMachineName())
+      ->setLabel($this->randomString())
+      ->setGroupType($this->group1->getEntityTypeId())
+      ->setGroupBundle($this->groupBundle)
+      ->setIsAdmin(TRUE)
+      ->save();
```

Comment:
Indeed. This reminds me that we should allow setting is-admin in the default roles definition